style.css

```diff
@@ -1,3 +1,20 @@
+body {
+  font-size: 20px;
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  padding: 12vw;
+  background-color: #cef;
+}
+h1, p, a {
+  text-align: center;
+}
```

Comment:
Please make sure to also center `h2` so that the smaller heading is centered

Current: 
![image](https://user-images.githubusercontent.com/92315435/137599034-3606e71b-d0d0-4790-9b18-ba58ca1c6baf.png)